mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -1473,6 +1473,26 @@ func.func @extract_scalar_from_vec_0d_index(%arg0: vector<index>) -> index {
 //       CHECK:   %[[T3:.*]] = builtin.unrealized_conversion_cast %[[T2]] : i64 to index
 //       CHECK:   return %[[T3]] : index
 
+
+// -----
+
+func.func @extract_scalar_from_vec_2d_f32_dynamic_idxs_compile_time_const(%arg : vector<32x1xi32>) -> i32 {
+  %0 = arith.constant 0 : index
+  %1 = vector.extract %arg[%0, %0] : i32 from vector<32x1xi32>
+  return %1 : i32
+}
+
+// Compile-time if the indices of extractOp if constants, the constants will be collapsed,
+// the constants are folded away, hence the lowering works.
```

Comment:
[nit] Small grammar fix
```suggestion
// At compile time, since the indices of extractOp are constants, they will be collapsed and folded away; therefore, the lowering works.
```